contracts/interfaces/IBondingMinter.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+interface IBondingMinter {
+    event StartBonding(uint startTimestamp, uint endTimestamp);
+    event Mint(address indexed tokenId, address indexed user);
+    event MintData(address indexed asset, uint allocation, uint release, string bondType);
+
+    function startBonding() external;
```

Comment:
What does is this supposed to do? In general I'd say we need comments for all the methods in the interface to avoid confusion, this should be a self-explanatory spec which is self evident without any conversations with any of the developer